android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java

```diff
@@ -266,7 +282,7 @@ public void onUrlDeviceDiscoveryUpdate() {
       @Override
       public void run() {
         for (PwPair pwPair : mPwCollection.getGroupedPwPairsSortedByRank(
-            Utils.newDistanceComparator())) {
+            new Utils.PwPairComparator())) {
```

Comment:
I feel like this should have a little more descriptive name...like PwPairRankComparator